src/screens/ExchangeModal.tsx

```diff
@@ -562,7 +616,7 @@ export default function ExchangeModal({
           inputTokenName: inputCurrency?.name || '',
           inputTokenSymbol: inputCurrency?.symbol || '',
           isHighPriceImpact: debouncedIsHighPriceImpact,
-          liquiditySources: tradeDetails?.protocols || [],
+          liquiditySources: JSON.stringify(tradeDetails?.protocols || []),
```

Comment:
@estebanmino @jxom Wasn't 100% sure on this one but it seems like the tracking can only have values that are json serializable and so this felt like the cleanest way of doing this